### DISCUSSION TITLE
I got the error: ModuleNotFoundError: No module named 'esm'

### DISCUSSION THREAD

**Discussion by lamnnguyen** [brianhie/efficient-evolution#20] [General]
Hi Brian,
Congratulation on this great work. 
I am new to Python and I got the message: ModuleNotFoundError: No module named 'esm'
How can I fix it? sorry for this very basic question. 
Thank you so much.

**Comment by brianhie** [comment]
Hi @lamnnguyen, you just need to install the esm module by running
```bash
pip install fair-esm
```
from the command line (https://github.com/facebookresearch/esm#getting-started-with-this-repo-)

**Reply by lamnnguyen** [reply]
Hi Brian,
Thank you so much. It works for me now. Really appreciate.
